templates/onebook.html

```diff
@@ -0,0 +1,15 @@
+<div class="book">
+  <div>
+    <img class="book-cover" ng-src="{{obc.book.image}}">
+  </div>
+  <form class="editing" ng-submit="obc.submit(obc.book)">
+    <h3 class="title"><span ng-hide="edit">{{obc.book.title}}</span><input type="text" name="title" ng-show="edit" ng-model="obc.editBook.title"></h3>
```

Comment:
These are each a lot of things on one line. I try never to have more than one tag in a line; I'd break this up into like 4 lines.